scipy/stats/_entropy.py

```diff
@@ -143,7 +144,9 @@ def entropy(pk: np.typing.ArrayLike,
         S /= np.log(base)
     return S
 
-
+@_axis_nan_policy_factory(
+    lambda x: x, n_outputs=1, result_to_tuple=lambda x: (x,)
```

Comment:
```suggestion
    lambda x: x, n_outputs=1, result_to_tuple=lambda x: (x,)
```
At some point, if you wouldn't mind submitting a PR to change the default `result_to_tuple` to the identity function, I would be happy to review that. The only reason it is not is because the decorator was originally developed for hypothesis tests.